README.md

```diff
@@ -193,24 +199,73 @@ index.search('prince')
 All the supported options are described in [this documentation section](https://docs.meilisearch.com/references/search.html#search-in-an-index).
 
 ```ruby
-index.search('prince', { limit: 1 })
+index.search('prince', { limit: 1, attributesToHighlight: '*' })
 ```
 
 ```json
 {
     "hits": [
         {
-            "id": 456,
-            "title": "Le Petit Prince"
+            "title": "Le Petit Prince",
+            "_formatted": {
+                "title": "Le Petit <em>Prince</em>"
+            }
         }
     ],
     "offset": 0,
     "limit": 1,
-    "processingTimeMs": 10,
+    "processingTimeMs": 0,
     "query": "prince"
 }
 ```
 
+## ⚙️ Development Workflow
+
+If you want to contribute, this sections describes the steps to follow.
+
+Thank you for your interest in a MeiliSearch tool! ♥️
+
+### Install dependencies
+
+```bash
+$ bundle install
```

Comment:
Does this install the development dependencies as well?

Reply:
Yes, it installs all by default. You can specify a `--without` or `--with` option to be more accurante.
https://bundler.io/bundle_install.html